GPT/beta-commands/beta-cursorless.talon

```diff
@@ -14,3 +14,8 @@ app: vscode
     text_list = user.cursorless_get_text_list(cursorless_target)
     default_destination = user.cursorless_create_destination(cursorless_target)
     user.gpt_dynamic_request_cursorless(user.text, text_list, cursorless_destination or default_destination)
+
+model blend <user.cursorless_target> [<user.cursorless_destination>]$:
+    target_text = user.cursorless_get_text_list(cursorless_target)
+    result = user.model_blend(target_text, edit.selected_text())
+    user.cursorless_or_paste_helper(cursorless_destination or 0, result)
```

Comment:
This doesn't look right to me. If you supply a destination, it uses selected text in prompt context, but then places the output at destination. I had thought the idea was for destination to be used for both prompt context and output if destination is supplied

Reply:
Yeah, you're right. I was having terrible getting the text from the cursorless destination. Is there an api for that? The alternative could be to use a cursorless target for the destination and use the api to make that into a destination, which should work. What do you think?

Reply:
Yeah I think I'd do that for now and only support "to"

Reply:
Sounds good. I'll plan to do that next chance I get.